agtern/pipelines/frontend/api.py

```diff
@@ -1,8 +1,11 @@
 """Pre-MVP: This file calls functions in agtern.backend.server to retrieve data.
 Post-MVP: This file will provide functions that will query the AgTern web server."""
 
+from typing import List
+
 from ..backend import get_all_internships
```

Comment:
```suggestion
import ..backend as backend
```
If you use an alias, there won't be a function name conflict.